source/blender/render/hydra/viewportEngine.cc

```diff
@@ -8,332 +8,37 @@
 #include <pxr/usd/usdGeom/camera.h>
 
 #include "BLI_math_matrix.h"
+#include "DNA_camera_types.h"
 
 #include "glog/logging.h"
 
 #include "viewportEngine.h"
+#include "camera.h"
 #include "utils.h"
 
 using namespace std;
 using namespace pxr;
 
 namespace blender::render::hydra {
 
-struct CameraData {
-  static CameraData init_from_camera(BL::Camera &b_camera, float transform[4][4], float ratio, float border[2][2]);
-  static CameraData init_from_context(BL::Context &b_context);
-
-  pxr::GfCamera export_gf(float tile[4]);
-
-  BL::Camera::type_enum mode;
-  float clip_range[2];
-  float focal_length = 0.0;
-  float sensor_size[2];
-  float transform[4][4];
-  float lens_shift[2];
-  float ortho_size[2];
-  tuple<float, float, int> dof_data;
-};
-
 struct ViewSettings {
```

Comment:
Mind moving ViewSettings to its own file?

Reply:
Probably later. Currently ViewSettings is pretty close to ViewportEngine only